common/lib/xmodule/xmodule/modulestore/tests/test_assetstore.py

```diff
@@ -423,13 +424,53 @@ def check_asset_values(assets, orig):
 
                 # Check 'em.
                 for asset_type, asset_list in (
-                    ('different', differents),
-                    ('vrml', vrmls),
-                    ('asset', regular_assets),
+                    ('different', self.differents),
+                    ('vrml', self.vrmls),
+                    ('asset', self.regular_assets),
+                ):
+                    assets = store.get_all_asset_metadata(course.id, asset_type)
+                    self.assertEquals(len(assets), len(asset_list))
+                    self._check_asset_values(assets, asset_list)
+
+                self.assertEquals(len(store.get_all_asset_metadata(course.id, 'not_here')), 0)
+                self.assertEquals(len(store.get_all_asset_metadata(course.id, None)), 4)
+
+                assets = store.get_all_asset_metadata(
+                    course.id, None, start=0, maxresults=-1,
```

Comment:
I just meant that since `maxresults=-1` is the default, you don't necessarily need to include it in the argument list. I'm also a bit leery about using `-1` as the magic flag for "all results".